samples/browser-extension/test/test.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Extension Test Page</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+
+    <script>
+        const xpocLink = 'xpoc://christianpaquin.github.io!';
```

Comment:
I suggest replacing the xpoc link with `xpoc://raw.githubusercontent.com/microsoft/xpoc-framework/main/samples/tests!`, and adding this page to the corresponding manifest; this way the page will show valid links